src/browser-lib/avo/getAvoBridge.ts

```diff
@@ -50,8 +50,21 @@ const getAvoBridge = ({ posthog }: AnalyticsServices) => {
     posthog.track(eventName, eventProperties);
   };
 
+  const identify: CustomDestination["identify"] = (userId) => {
+    posthog.identify(userId, {});
+  };
+
+  const setUserProperties: CustomDestination["setUserProperties"] = (
+    userId,
+    properties,
+  ) => {
+    posthog.identify(userId, properties);
+  };
```

Comment:
This appears to be the first time that we're setting user properties through Avo and this wasn't supported by the existing analytics setup. Defining these functions ensures that the new methods route analytics calls to the correct place.

Reply:
I'm not 100% but it looks like the `withQueue` method that posthog is used with implements the identify method?

Reply:
That's what this eventually calls 🤣 took me a while to burrow through the abstractions to grok.

<img width="964" alt="Screenshot 2024-10-11 at 09 58 21" src="https://github.com/user-attachments/assets/e20a06f5-80db-40ea-bb4f-84bf901ff8a4">

`getAvoBridge` returns an interface that Avo's codegen expects. All it does is call methods on the posthog abstraction you mentioned. Without these wrappers in the bridge it was blowing up in the codegen.

Reply:
Ohhh... yeah I was sure i'd seen identify somewhere but couldn't parse what was going on at all on inspection 😂 fair enough!